src/wasm/rlottiewasm.cpp

```diff
@@ -41,6 +41,50 @@ class __attribute__((visibility("default"))) RlottieWasm {
 
         return val(typed_memory_view(mWidth * mHeight * 4, mBuffer.get()));
     }
+
+    void setFillColor(std::string key, float r, float g, float b){
+        mPlayer->setValue<rlottie::Property::FillColor>(key, rlottie::Color(r, g, b));
+    }
+  
+    void setFillOpacity(std::string key, float opacity) {
+        mPlayer->setValue<rlottie::Property::FillOpacity>(key, opacity);
+    }
+
+    void setStrokeColor(std::string key, float r, float g, float b){
+        mPlayer->setValue<rlottie::Property::StrokeColor>(key, rlottie::Color(r, g, b));
+    }
+
+    void setStrokeOpacity(std::string key, float opacity) {
+        mPlayer->setValue<rlottie::Property::StrokeOpacity>(key, opacity);
+    }
+
+    void setStrokeWidth(std::string key, float width) {
+        mPlayer->setValue<rlottie::Property::StrokeWidth>(key, width);
+    }
+
+    void setTrAnchor(std::string key, float x, float y){
+        mPlayer->setValue<rlottie::Property::TrAnchor>(key, rlottie::Point(x, y));
+    }
+
+    void setTrPosition(std::string key, float x, float y){
+        mPlayer->setValue<rlottie::Property::TrPosition>(key, rlottie::Point(x, y));
+    }
+
+    void setTrScale(std::string key, float w, float h){
+        mPlayer->setValue<rlottie::Property::TrScale>(key, rlottie::Size(w, h));
+    }
+
+    void setTrRotation(std::string key, float degree) {
+        mPlayer->setValue<rlottie::Property::TrRotation>(key, degree);
+    }  
+
+    void setTrOpacity(std::string key, float opacity) {
+        mPlayer->setValue<rlottie::Property::TrOpacity>(key, opacity);
+    }
+
+    std::string getBasicResource(){
```

Comment:
what the need of this Api ?

Reply:
This API was created to use the initial resource within 'rlottiewasm.cpp' in the preview.
Do you want to handle this in the preview?

Thank you for your reviews :D